### PR TITLE
Correction in the computation of sigma for the next population in CMA-ES.

### PR DESCRIPTION
**Pull request by gaurav-singh1998**
I found a minor bug in the computation of sigma for the next population and fixed it.

### CODE REVIEW

File: include/ensmallen_bits/cmaes/cmaes_impl.hpp [hunk 1]
```diff
@@ -216,8 +216,7 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
     }
 
     const ElemType psNorm = arma::norm(ps[idx1]);
-    sigma(idx1) = sigma(idx0) * std::pow(
-        std::exp(cs / ds * psNorm / enn - 1), 0.3);
+    sigma(idx1) = sigma(idx0) * std::exp(cs / ds * psNorm / enn - 1);
```
Comment (favre49) [inline include/ensmallen_bits/cmaes/cmaes_impl.hpp]: Should this not be 
```suggestion
    sigma(idx1) = sigma(idx0) * std::exp(cs / ds *( psNorm / enn - 1));
```

Comment (gaurav-singh1998) [reply include/ensmallen_bits/cmaes/cmaes_impl.hpp]: Oh! thanks for pointing that out @favre49

File: include/ensmallen_bits/cmaes/cmaes_impl.hpp [hunk 2]
```diff
@@ -216,8 +216,7 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
     }
 
     const ElemType psNorm = arma::norm(ps[idx1]);
-    sigma(idx1) = sigma(idx0) * std::pow(
-        std::exp(cs / ds * psNorm / enn - 1), 0.3);
+    sigma(idx1) = sigma(idx0) * std::exp(cs / ds *( psNorm / enn - 1));
```
Comment (favre49) [inline include/ensmallen_bits/cmaes/cmaes_impl.hpp]:
```suggestion
    sigma(idx1) = sigma(idx0) * std::exp(cs / ds * ( psNorm / enn - 1));
```

**Comment by favre49** [comment]
I'll wait on the tests before approval. Good catch!

**Comment by gaurav-singh1998** [comment]
Hi, @favre49 why the `momentum_sgd_test.cpp` is failing although it has nothing do with this PR?

**Comment by gaurav-singh1998** [comment]
Hi, @favre49 also in the implementation I noticed that for `sigma` [here](https://github.com/mlpack/ensmallen/blob/master/include/ensmallen_bits/cmaes/cmaes_impl.hpp#L80) and for `mPositon` [here](https://github.com/mlpack/ensmallen/blob/master/include/ensmallen_bits/cmaes/cmaes_impl.hpp#L100) a vector of size 3 has been allocated but why do we need a vector of size 3 when only 2 values are needed i.e. for the current population and the next population?

**Comment by favre49** [comment]
You're right, I don't think it needs to be of size 3. Do you mind changing that?

**Comment by gaurav-singh1998** [comment]
Okay, @favre49 I will make those changes in this PR itself. Thanks.

**Comment by zoq** [comment]
Can you update the `HISTORY.md` file.

**Comment by gaurav-singh1998** [comment]
Hi, @zoq it is done. Thanks.

**Comment by favre49** [comment]
Alright, I think we can merge this now. Thanks!

**Pull request merged**